public/src/components/epicTests/epicTestsList.tsx

```diff
@@ -176,10 +168,13 @@ class EpicTestsList extends React.Component<EpicTestListProps, EpicTestsListStat
             })}
           </List>
 
-          <EpicTestEditor
-            test={this.state.selectedTestName ? this.props.tests.find(test => test.name === this.state.selectedTestName) : undefined}
+          {this.props.tests.map(test => (<EpicTestEditor
```

Comment:
Perhaps for a separate PR, but now that `selectedTestName` has been moved up into `EpicTestsForm`, the creation of the `EpicTestEditor` components can happen in `EpicTestsForm` as well.

Reply:
Yes, good idea - will address in a separate PR.